src/logic/operate.js

```diff
@@ -20,7 +20,7 @@ export const operate = (one, two, operation) => {
   }
 
   if (operation === '%') {
-    result = Big(one).mod(Big(two));
+    result = (Big(100).div(Big(one).plus(Big(two)));
```

Comment:
I guess it's should be something like this;
we are trying to get the percentage of the first number, so by doing `big(one) / 100` is ok

```
let numberOne = Big(firstNumber)
return numberOne / 100;
```